debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -728,6 +731,21 @@ private static void dumpClassJ9ExtraModifiers(PrintStream out, long accessFlags)
 			out.append("(sealed) ");
 	}
 
+	private static void dumpMethodJ9Modifiers(PrintStream out, UDATA modifiers, UDATA extraModifiers) {
+		if (modifiers.allBitsIn(J9JavaAccessFlags.J9AccMethodCallerSensitive)) {
+			out.append(" (@CallerSensitive)");
+		}
+		if (modifiers.allBitsIn(J9JavaAccessFlags.J9AccMethodFrameIteratorSkip)) {
+			out.append(" (@FrameIteratorSkip)");
+		}
+		if (modifiers.allBitsIn(J9CfrClassFile.CFR_METHOD_EXT_HAS_SCOPED_ANNOTATION)) {
+			out.append(" (@Scoped)");
+		}
+		if (modifiers.allBitsIn(J9CfrClassFile.CFR_METHOD_EXT_NOT_CHECKPOINT_SAFE_ANNOTATION)) {
+			out.append(" (@NotCheckpointSafe)");
```

Comment:
Two questions:
* Are the parentheses necessary?
* Should the modifiers be listed in alphabetical order?